### DISCUSSION TITLE
Change "reply-to" for Airflow email notifications?

### DISCUSSION THREAD

**Discussion by match-gabeflores** [apache/airflow#30684] [Q&A]
We have email notifications set up for Airflow. When a dag fails, we get an email alert but the email from is set to : `airflow@mycompany.com` 

We would like to modify the `reply-to` in these cases to be `person@mycompany.com`. Is this possible? 

Note, we are *not* using EmailOperator or any user-defined send_email/oncallback functions. We just set up default smtp and have the built-in alerting when tasks fail. 
It seems EmailOperator, you can pass in "custom_headers" with reply-to argument ([link][1]) but we are not using emailoperator. 

  [1]: https://stackoverflow.com/questions/73768320/send-reply-to-param-in-airflow-emailoperator

Airflow 2.2.5

**Comment by potiuk** [comment]
```
# Configuration email backend and whether to
# send email alerts on retry or failure
# Email backend to use
email_backend = airflow.utils.email.send_email_smtp
```

You can write your own python method (for example in local settings) that will wrap the `email_dend_email_smtp` and add the cusotm headers - then you can point to that custom methd of yours in the configuration.
